src/js/Controller/Client/OpenUrl.js

```diff
@@ -0,0 +1,31 @@
+import FormService from '@js/Services/FormService';
```

Comment:
We don't need the client to open an url.
[tabs.update](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/update) or [tabs.create](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/create) can be used for this.